lib/util/git/index.ts

```diff
@@ -480,11 +481,26 @@ export async function syncGit(): Promise<void> {
     }
     logger.warn({ err }, 'Cannot retrieve latest commit');
   }
-  config.currentBranch =
-    config.currentBranch ??
-    config.defaultBranch ??
-    (await getDefaultBranch(git));
-  delete getCache()?.semanticCommits;
+  config.currentBranch = config.currentBranch || (await getDefaultBranch(git));
```

Comment:
Are you sure these changes should be made? It looks like the code base here has changed since the original PR. The original PR did not change these lines.

Reply:
I think you're right, copied too much code from the old PR. I restored these lines.